cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,275 @@
+
+Below are the runtime variables that can be used in the fireboltCall object.
+
+- **attribute:** Attribute is a field name of the `runtime` environment variable that holds the value of the method name. For example, if the method name is `closedCaptions.setEnabled`, the attribute value will be `enabled`.
+- **value:** Value is a field name of the `runtime` environment variable that holds the value used for to set the value or for validation.
+
+**Note:** In order to use the runtime variables in the fireboltCall object, the `resolveAtRuntime()` function should be used.
+
+## ResolveAtRuntime Function
+
+**ResolveAtRuntime** is a global function utilized by the JS files. This function is used to dynamically resolve the values during the test case execution. It is used to replace the placeholders in the fireboltCall object with the actual values from the `runtime` environment variable.
+
+- **Input Types**: `resolveAtRuntime` accepts either a string or an array of strings as input.
+
+  - **String Input with Pattern**: When a string contains the '{{' pattern (e.g., `resolveAtRuntime("manage_closedcaptions.set{{attribute}}")`), the function replaces the pattern with the actual value from the `runtime` environment variable.
+  - **String Input without Pattern**: If the string does not contain the '{{' pattern (e.g., `resolveAtRuntime("value")`), the function directly returns the value from the `runtime` environment variable.
+  - **Array of Strings**: When an array of strings is provided, the function iterates over each string, replacing the patterns with the actual values from the `runtime` environment variable.
+
+- **Options:** The function has the support to accespts the optional parameter `uppercaseFirstChar` or `lowercaseFirstChar` to convert the first character of the resolved value to uppercase or lowercase. This parameter is separated by a dot from the attribute name. This parameter is used to convert the attribute value to the correct format for the method name. For example, `resolveAtRuntime('manage_closedcaptions.set{{attribute.uppercaseFirstChar}}')`.
+
+This process ensures that all necessary values are dynamically resolved during the test case execution.
+
+### Examples:
+
+Assuming runtime environment variable having below details
+
+```
+runtime: {
+ attribute: 'fontSize',
+ value: 1.5,
+}
+```
+
+#### Example 1:
+
+```
+resolveAtRuntime(["result.{{attribute}}", "result.styles.{{attribute}}"])
+returns: ['result.fontSize', 'result.styles.fontSize']
+```
+
+#### Example 2:
+
+```
+resolveAtRuntime("manage_closedcaptions.set{{attribute.uppercaseFirstChar}}")
+returns: "manage_closedcaptions.setFontSize"
+```
+
+#### Example 3:
+
+```
+resolveAtRuntime("value")
+returns: 1.5
+```
+
+## Usage
+
+The dynamic calls Glues can be used in test cases along with fireboltCall dynamic objects.
+
+### Below examples shows the how fireboltCall object and testcase can be written using dynamic glues.
+
+#### Example 1:
```

Comment:
Give some description to each example/test case. Exactly what this test case will do. How the values are used in dynamic objects. etc

Reply:
Added the description for examples

Reply:
addressed the comments